src/displayobjects/rectangle.js

```diff
@@ -16,13 +16,18 @@
 					y = this.abs_y - origin.y;
 				
 				canvas.beginPath();
+
+				canvas.rect(x, y, this.width, this.height);
 				
 				// Do fill if a color is specified
 				if (this.fill !== "") {
 					canvas.fillStyle = this.fill;
-					canvas.fillRect(x, y, this.width, this.height);
+					canvas.fill();
 				}
-				
+
+				// // Save canvas (for clipping after draw the stroke)
+				// canvas.save();
```

Comment:
Why is this here and commented out? Will the stroke interfere with the clipping?